docs/tutorials/api/datasets.ipynb

```diff
@@ -0,0 +1,877 @@
+{
```

Comment:
<p>Maybe a subtitle to explain you are creating an empty dataset first</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>